components/form/demo/advanced-search-form.md

```diff
@@ -114,7 +114,7 @@ ReactDOM.render(
   border-radius: 6px;
 }
 
-/* becasue the label width is not definite, you need to adjust it to center by yourself */
+/* because the label width is not fixed, you may need to adjust it */
```

Comment:
In this case, I use this style to center the form item, 
but be because the label width is not fixed,
anyone want to center the element, need to modify the left position,
so I think you need to mention that?